Cargo.toml

```diff
@@ -78,7 +78,7 @@ num-bigint = "0.4.5"
 num-traits = "0.2.19"
 nunny = "0.2.1"
 once_cell = "1.19.0"
-paladin-core = "0.4.2"
+paladin-core = { git = "https://github.com/0xPolygonZero/paladin.git", branch = "main" }
```

Comment:
@muursh Are we going to release new paladin version or we are targeting github head?

Reply:
afaik we will be releasing a new version

Reply:
Yeah we should release a new version. @temaniarpit27 is everything ready to go for a new release?

Reply:
Did we get actual benchmark measurements of this approach?

Reply:
no we haven't benchmaked it yet